lib/src/code_generator/objc_built_in_functions.dart

```diff
@@ -144,11 +166,19 @@ class ObjCBuiltInFunctions {
 
     final objType = PointerType(objCObjectType).getCType(w);
     s.write('''
-class _ObjCWrapper {
+class _ObjCWrapper implements ${w.ffiLibraryPrefix}.Finalizable {
```

Comment:
I think we also want a `release` method that 

1. sets `_isReleased` to true
2. cancels the finalizer
3. decrements the ref-count

And then throw on any method invocations if `_isReleased` is true.

(This only applies for `release = true` constructor calls, so we should also save `_release` in the object.)

Reply:
Done